src/Dashboard/ApiControllers/FunctionsController.cs

```diff
@@ -341,6 +341,7 @@ private InvocationLogViewModel CreateInvocationEntry(RecentInvocationEntry entry
             metadataSnapshot.EndTime = entry.EndTime;
             metadataSnapshot.Succeeded = entry.Succeeded;
             metadataSnapshot.Heartbeat = entry.Heartbeat;
+            metadataSnapshot.HeartbeatExpiredTime = entry.HeartbeatExpiredTime;
```

Comment:
You can't roll this in to the existing Heartbeat hanging off of the snapshot because that is the legacy blob stuff? It will be somewhat confusing to have both of these properties floating around in the codebase.